resources/web/wwi/nodes/WbCadShape.js

```diff
@@ -287,38 +297,49 @@ export default class WbCadShape extends WbBaseNode {
       17: ambient occlusion
     */
 
+    let materialPrefix;
+    if (this.isCollada || typeof webots.currentView.stream === 'undefined')
+      materialPrefix = ''; // for animations the texture isn't relative to the material but included in the 'textures' folder
```

Comment:
I think Collada should also have a prefix no?

Reply:
yep, my bad